src/rpm/headers/mod.rs

```diff
@@ -0,0 +1,13 @@
+mod header;
+mod lead;
+mod types;
+
+pub use header::*;
+pub use lead::*;
+pub use types::*;
+
+#[cfg(feature = "signing-meta")]
+mod signature_builder;
```

Comment:
I think this should be just the module `signature`.  This module may contain more functions for header signatures in the future